python/ray/data/datasource/file_meta_provider.py

```diff
@@ -329,17 +329,17 @@ def prefetch_file_metadata(
             return _fetch_metadata(pieces)
 
 
-def _handle_read_s3_files_error(error: OSError, paths: Union[str, List[str]]) -> str:
-    # Specially handle AWS error when reading files, to give a clearer error message
-    # to avoid confusing users. The real issue is most likely that the AWS S3 file
-    # credentials have not been properly configured yet.
+def _handle_read_os_error(error: OSError, paths: Union[str, List[str]]) -> str:
     if "AWS Error [code 15]: No response body" in str(error):
+        # Specially handle AWS error when reading files, to give a clearer error
+        # message to avoid confusing users. The real issue is most likely that the AWS
+        # S3 file credentials have not been properly configured yet.
```

Comment:
This is just move the original comment literally without any change. Feel make more sense to comment inside the `if` branch, as we plan to extend the function to apply to other file system such as GCS in the future.